synapse/util/ldap_auth_provider.py

```diff
@@ -237,13 +237,14 @@ def _ldap_simple_bind(self, server, localpart, password):
                 base=self.ldap_base
             )
             conn = ldap3.Connection(server, bind_dn, password,
-                                    authentication=ldap3.AUTH_SIMPLE)
+                                    authentication=ldap3.SIMPLE)
             logger.debug(
                 "Established LDAP connection in simple bind mode: %s",
                 conn
             )
 
             if self.ldap_start_tls:
+                conn.bind()
```

Comment:
As I understand it that would mean to authenticate against the ldapd in plaintext.